src/styles/_typography.scss

```diff
@@ -0,0 +1,4 @@
+$font: Roboto, sans-serif;
+$main_font-weight: 500;
+$accent_font-weight: 700;
+$secondary_font-weight: 400;
```

Comment:
Default value, so you can just remove it everywhere
```suggestion
```

Reply:
fixed